guidelines/releases-guidelines/02-release-process.md

```diff
@@ -44,76 +70,101 @@ See the `TestBucReleases` test defined in `development/tools/jobs/kyma/service_b
+>**NOTE:** This point applies for new release candidates such as rc1, rc2 and the final version.
+
+1. Ensure that the `prow/RELEASE_VERSION` file from the `test-infra` repository on a release branch contains the correct version to be created. The file should contain a release version following the `{A}.{B}.{C}` or `{A}.{B}.{C}-rc{D}` format, where `A`,`B`, `C`, and `D` are numbers. If you define a release candidate version, a pre-release is created.  
 
 	Make sure the `RELEASE_VERSION` file includes just this single line:  
-	\`\`\`bash
-	echo -n {RELEASE\_VERSION} \> prow/RELEASE\_VERSION
-	\`\`\\\`
+	```bash
+	echo -n {RELEASE_VERSION} > prow/RELEASE_VERSION
+	```
 
-3. Push the branch to the `test-infra` repository.
+1. Push the branch to the `test-infra` repository.
 
-4. Create a PR to `test-infra/release-x.y`. This triggers the pre-release job for `watch-pods`. 
+1. Create a PR to `test-infra/release-x.y`. This triggers the pre-release job for `watch-pods`. 
 
 	> **NOTE:** To trigger the `watch-pods` build without introducing any changes, edit any file within the `test-infra` repository and create a pull request. You don't need to merge it as the job is triggered anyway. After a successful `watch-pods` image build, close the pull request.
 
-5. Update the `RELEASE_VERSION`file with the name of the next minor release candidate and merge the pull request to `master`. For example, if the `RELEASE_VERSION` on the `master` branch is set to `0.9.2`, then change the version to `1.0.0-rc1`.
+1. Update the `RELEASE_VERSION` file with the name of the next minor release candidate and merge the pull request to `master`. For example, if the `RELEASE_VERSION` on the `master` branch is set to `1.4.2`, then change the version to `1.5.0-rc1`.
+
 
-### kyma-project/kyma
+### Changes to Kyma repository
+
+#### Steps for new major/minor version
 
 1. Create a release branch in the `kyma` repository.
 
 	Do it only for major and minor releases.
-	The name of this branch should follow the `release-x.y` pattern, such as `release-0.9`.
+	The name of this branch should follow the `release-x.y` pattern, such as `release-1.4`.
 
 	```bash
 	git fetch upstream
 	git checkout -b release-{release_version} upstream/master
 	git push upstream release-{release_version}
 	```
 
-	> **NOTE:** If you push anything to the release branch, or if you rebase the branch, a new Github release is created.
+	> **NOTE:** If you push anything to the release branch (includes creating of the branch), or if you rebase the branch, a new Github release is created. This is because of the post-submit job `post-rel14-kyma-release-candidate` in `kyma-github-release.yaml` and only happens if you already merged the test-infra changes.
```

Comment:
Why don't we do this step before creating a release branch in test-infra or merging new release changes to master/test-infra? Because when a developer reaches this step, a false release will anyway get created as there is no point of return. All we need to make sure is not to push the changes in test-infra before creating/rebasing(rarely needed) the release branch of Kyma repo. This can be done on the day of the release rc1. And the preparation work on test-infra can start before(except `git push ....`)

Reply:
I incorporated your feedback. Please see the new commit